README.md

```diff
@@ -44,14 +44,16 @@ Or use the power of Vue.js within a google map like this:
 <template>
   <gmap-map
     :center="center"
+    @center_changed="center=$event"
```

Comment:
I feel that this should be `camelCase` like the rest of the project, both here and elsewhere in this PR